### ISSUE TITLE
MAC: Passport initialisation

### ISSUE THREAD

**Issue by Sanselll** [immutable/unity-immutable-sdk#159]
Plugin v1.1.0, Unity 2022.3.15
 Passport initialisation doesn't work in Mac build, works well in Mac Editor but doesn't work in the build.
 Having this in logs:
```
 Passport:Initialising Passport...
 Passport:Waiting for ready signal...
```

**Comment by nattb8** [comment]
Hi @Sanselll,

When building, what do you see in the Console? Our SDK includes a [post-process](https://github.com/immutable/unity-immutable-sdk/blob/main/src/Packages/Passport/Editor/PassportPostprocess.cs) script that copies certain files required to run the SDK. If the files are copied correctly to the build, you should see the message "Successfully copied Passport files" in the console output.

![image](https://github.com/immutable/unity-immutable-sdk/assets/16816284/190b9d6c-5943-4243-b32c-225621caa204)

**Comment by Sanselll** [comment]
Hi @nattb8 
No errors 
<img width="983" alt="Screenshot 2024-02-01 at 08 46 28" src="https://github.com/immutable/unity-immutable-sdk/assets/3996665/aa721031-853c-4506-8f10-0a74b56da9fd">

**Comment by Sanselll** [comment]
The issue is that webview can't recognize the path tho to the "index.html" if the app name has whitespace.
Steps to replicate:
_1. build an app with the name "Sample X.app"_

**Possible fix:**
  ```
 string escapedPath = filePath.Replace(" ", "%20");
            webViewObject.LoadURL(escapedPath);
```

**Comment by nattb8** [comment]
Thanks @Sanselll. I've added the fix to [v1.3.0](https://github.com/immutable/unity-immutable-sdk/releases/tag/v1.3.0).

**Comment by BestStream** [comment]
@nattb8 Are you sure you solved the problem? Because I have a similar problem that reproduces on version 1.30.0
